docs/azure/includes/dotnet-all.md

```diff
@@ -348,7 +348,7 @@
 | Event Hubs - Event Processor | NuGet [4.3.2](https://www.nuget.org/packages/Microsoft.Azure.EventHubs.Processor/4.3.2) |  | GitHub [4.3.2](https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.EventHubs.Processor_4.3.2/sdk/eventhub/Microsoft.Azure.EventHubs.Processor/) |
 | Event Hubs - Service Fabric Processor | NuGet [0.5.4](https://www.nuget.org/packages/Microsoft.Azure.EventHubs.ServiceFabricProcessor/0.5.4) |  | GitHub [0.5.4](https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.EventHubs.ServiceFabricProcessor_0.5.4/sdk/eventhub/Microsoft.Azure.EventHubs.ServiceFabricProcessor/) |
 | Face | NuGet [2.8.0-preview.3](https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Vision.Face/2.8.0-preview.3) |  | GitHub [2.8.0-preview.3](https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.CognitiveServices.Vision.Face_2.6.0-preview.1/sdk/cognitiveservices/Vision.Face) |
-| Feature Management | NuGet [2.5.1](https://www.nuget.org/packages/Microsoft.FeatureManagement/2.5.1) | [docs](https://learn.microsoft.com/dotnet/api/microsoft.featuremanagement?view=azure-dotnet) |  |
+| Feature Management | NuGet [2.5.1](https://www.nuget.org/packages/Microsoft.FeatureManagement/2.5.1) | [docs](https://learn.microsoft.com/en-us/dotnet/api/microsoft.featuremanagement?view=azure-dotnet) |  |
```

Comment:
Remove locale.

Ping @alexwolfmsft Can you fix the source so the locale doesn't get added back into the next PR for this?

```suggestion
| Feature Management | NuGet [2.5.1](https://www.nuget.org/packages/Microsoft.FeatureManagement/2.5.1) | [docs](https://learn.microsoft.com/dotnet/api/microsoft.featuremanagement?view=azure-dotnet) |  |
```